<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>41</y>
      <width>311</width>
      <height>309</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="serverAddressComboBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="serverAddressLineEdit">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="connectToServerPushButton">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="serverPortSpinBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="disconnectFromServerPushButton">
       <property name="text">
        <string>disconnect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comPortComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="connectComPortPushButton">
       <property name="text">
        <string>connect COM</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="initSpectraPushButton">
       <property name="text">
        <string>init spectra</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="eyesFilePushButton">
       <property name="text">
        <string>eyes file</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="comPortSendPushButton">
       <property name="text">
        <string>send</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="fullDataCheckBox">
       <property name="text">
        <string>full data</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="comPortSpinBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="startPushButton">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="startRealTimePushButton">
       <property name="text">
        <string>start real-time</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="endPushButton">
       <property name="text">
        <string>end</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="askPushButton">
       <property name="text">
        <string>ask server</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ampl</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="amplSpinBox"/>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>theta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="thetaSpinBox"/>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>beta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="betaSpinBox"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="comPortSendTwoPushButton">
       <property name="text">
        <string>send 2</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="comPortSendOnePushButton">
       <property name="text">
        <string>send 1</string>
       </property>
       <property name="shortcut">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pauseSetPushButton">
       <property name="text">
        <string>set pause</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="pauseUnsetPushButton">
       <property name="text">
        <string>unset pause</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
